thoth/common/openshift.py

```diff
@@ -526,6 +540,43 @@ def _transform_cronjob_to_job(item: Dict[str, Any]) -> Dict[str, Any]:
         job_spec["metadata"]["generateName"] = item["metadata"]["name"] + "-"
         return job_spec
 
+    def _submit_workflow(
+        self,
+        wf: argo.models.V1alpha1Workflow,
+        namespace: Optional[str] = None,
```

Comment:
Let's have `namespace` required for this low level API for callee.

Reply:
Ack